### ISSUE TITLE
Installation hang for two hours

### ISSUE THREAD

**Issue by zzxwill** [keptn/keptn#3054]
Your issue may already be reported! Please search on the [issue tracker](https://github.com/keptn/keptn/issues) before creating a new one.

**Environment**

* Client OS (e.g., Linux, OSX, Windows): Linux
* Keptn Version (`keptn version`):0.7.3
* Kubernetes Cloud Provider (e.g., GKE, AKS): Alibaba Kubernetes Service (ACK)
* Kubernetes version (`kubectl version`): 
Client Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.3", GitCommit:"2e7996e3e2712684bc73f0dec0200d64eec7fe40", GitTreeState:"clean", BuildDate:"2020-05-20T12:52:00Z", GoVersion:"go1.13.9", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"18+", GitVersion:"v1.18.8-aliyun.1", GitCommit:"d2f5a0f", GitTreeState:"", BuildDate:"2020-10-23T06:48:31Z", GoVersion:"go1.13.15", Compiler:"gc", Platform:"linux/amd64"}

**Affected Component**

* [ ] Docs
* [ ] CLI
* [ ] Kubernetes Integration
* [ ] Openshift Integration
* [ ] Helm
* [ ] Istio
* [ ] Bridge
* [ ] Gatekeeper
* [ ] Datastore
* [ ] REST API
* [ ] eventbroker / distributor
* [ ] jmeter

**Describe the bug**

It hangs for two hours.
```
$ keptn install -v
Helm Chart used for Keptn installation: https://storage.googleapis.com/keptn-installer/keptn-0.7.3.tgz
Checking current Kubernetes context: kubectl config current-context
Result: kubernetes-admin-cde0fd53ee05a44d0af7c692116d25543


Please confirm that the provided cluster information is correct:
Checking current Kubernetes context: kubectl config current-context
Result: kubernetes-admin-cde0fd53ee05a44d0af7c692116d25543

Cluster: kubernetes-admin-cde0fd53ee05a44d0af7c692116d25543
Is this all correct? (y/n)
y
Installing Keptn ...
Start upgrading Helm Chart keptn in namespace keptn

```

mongoDB pod issues.
```
➜  /Users/zhouzhengxi k logs -f pod/mongodb-datastore-d44948bb9-79sql mongodb-datastore  -n keptn
##########
branch: release-0.7.3
repository: https://github.com/keptn/keptn
commitlink: https://github.com/keptn/keptn/commit/569b8328cc3d48b527bd569d8b058ab8044fee7c
repolink: https://github.com/keptn/keptn/tree/569b8328cc3d48b527bd569d8b058ab8044fee7c
travisbuild: https://travis-ci.com/keptn/keptn/jobs/434768766
timestamp: 20201112.1602
##########
2021/02/03 03:37:30 Serving mongodb datastore at http://[::]:8080
^C
➜  /Users/zhouzhengxi k logs -f pod/mongodb-datastore-d44948bb9-79sql distributor -n keptn
##########
branch: release-0.7.3
repository: https://github.com/keptn/keptn
commitlink: https://github.com/keptn/keptn/commit/569b8328cc3d48b527bd569d8b058ab8044fee7c
repolink: https://github.com/keptn/keptn/tree/569b8328cc3d48b527bd569d8b058ab8044fee7c
travisbuild: https://travis-ci.com/keptn/keptn/jobs/434768765
timestamp: 20201112.1538
##########
Connecting to NATS server at nats://keptn-nats-cluster...
2021/02/03 05:43:29 listen tcp :10999: bind: address already in use
failed to create NATS connection: nats: no servers available for connection
```

**To Reproduce**

1. Run `keptn install -v`

**Expected behavior**

**Current behavior**

**Screenshots**

**Additional context**

**Comment by zzxwill** [comment]
```
➜  /Users/zhouzhengxi k get all -n keptn
NAME                                         READY   STATUS             RESTARTS   AGE
pod/api-gateway-nginx-fcb6bf855-75f9x        1/1     Running            0          137m
pod/api-service-bc546f4d7-9bb2p              1/1     Running            0          137m
pod/bridge-6d6b77b865-sk6n5                  1/1     Running            0          137m
pod/configuration-service-749fd6894f-hdnr6   0/2     Pending            0          137m
pod/eventbroker-go-59d9b46d69-gcr2l          1/1     Running            0          137m
pod/keptn-nats-cluster-0                     3/3     Running            0          137m
pod/lighthouse-service-dbdb488bb-wvzhb       2/2     Running            3          137m
pod/mongodb-67668d9dc7-th5hx                 0/1     Pending            0          137m
pod/mongodb-datastore-d44948bb9-gphc7        1/2     CrashLoopBackOff   1          100s
pod/remediation-service-7577fdbbb9-tnw54     1/2     CrashLoopBackOff   27         137m
pod/shipyard-service-59c484f5b6-rsng9        2/2     Running            3          137m

NAME                            TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)                                                 AGE
service/api-gateway-nginx       ClusterIP   172.21.2.57     <none>        80/TCP                                                  137m
service/api-service             ClusterIP   172.21.15.208   <none>        8080/TCP                                                137m
service/bridge                  ClusterIP   172.21.11.104   <none>        8080/TCP                                                137m
service/configuration-service   ClusterIP   172.21.5.203    <none>        8080/TCP                                                137m
service/event-broker            ClusterIP   172.21.0.196    <none>        80/TCP                                                  137m
service/keptn-nats-cluster      ClusterIP   None            <none>        4222/TCP,6222/TCP,8222/TCP,7777/TCP,7422/TCP,7522/TCP   137m
service/lighthouse-service      ClusterIP   172.21.9.241    <none>        8080/TCP                                                137m
service/mongodb                 ClusterIP   172.21.9.227    <none>        27017/TCP                                               137m
service/mongodb-datastore       ClusterIP   172.21.7.108    <none>        8080/TCP                                                137m
service/remediation-service     ClusterIP   172.21.0.192    <none>        8080/TCP                                                137m
service/shipyard-service        ClusterIP   172.21.9.167    <none>        8080/TCP                                                137m

NAME                                    READY   UP-TO-DATE   AVAILABLE   AGE
deployment.apps/api-gateway-nginx       1/1     1            1           137m
deployment.apps/api-service             1/1     1            1           137m
deployment.apps/bridge                  1/1     1            1           137m
deployment.apps/configuration-service   0/1     1            0           137m
deployment.apps/eventbroker-go          1/1     1            1           137m
deployment.apps/lighthouse-service      1/1     1            1           137m
deployment.apps/mongodb                 0/1     1            0           137m
deployment.apps/mongodb-datastore       0/1     1            0           137m
deployment.apps/remediation-service     0/1     1            0           137m
deployment.apps/shipyard-service        1/1     1            1           137m

NAME                                               DESIRED   CURRENT   READY   AGE
replicaset.apps/api-gateway-nginx-fcb6bf855        1         1         1       137m
replicaset.apps/api-service-bc546f4d7              1         1         1       137m
replicaset.apps/bridge-6d6b77b865                  1         1         1       137m
replicaset.apps/configuration-service-749fd6894f   1         1         0       137m
replicaset.apps/eventbroker-go-59d9b46d69          1         1         1       137m
replicaset.apps/lighthouse-service-dbdb488bb       1         1         1       137m
replicaset.apps/mongodb-67668d9dc7                 1         1         0       137m
replicaset.apps/mongodb-datastore-d44948bb9        1         1         0       137m
replicaset.apps/remediation-service-7577fdbbb9     1         1         0       137m
replicaset.apps/shipyard-service-59c484f5b6        1         1         1       137m

NAME                                  READY   AGE
statefulset.apps/keptn-nats-cluster   1/1     137m
```

`kubectl logs` didn't print out logs for failure pod `configuration-service-749fd6894f-hdnr6`

**Comment by christian-kreuzberger-dtx** [comment]
The installation hanging problem has already been solved in https://github.com/keptn/keptn/issues/2988 and will be fixed in 0.8.0.

the fact that the installation doesn't come up looks like a problem with your Kubernetes cluster, not Keptn.

```
pod/configuration-service-749fd6894f-hdnr6   0/2     Pending            0          137m
```

and

```
pod/mongodb-67668d9dc7-th5hx                 0/1     Pending            0          137m
```

indicate to me that there might be a problem with storage/persistent volumes.


However, the additional crashes
```
pod/mongodb-datastore-d44948bb9-gphc7        1/2     CrashLoopBackOff   1          100s
pod/remediation-service-7577fdbbb9-tnw54     1/2     CrashLoopBackOff   27         137m
```

I'm not 100% sure why.


For troubleshooting purposes, can you please tell us the output of
* `kubectl describe nodes`
* ` kubectl -n keptn get pvc`
* `kubectl -n keptn describe deployment configuration-service`
* `kubectl -n keptn describe deployment mongodb`

**Comment by christian-kreuzberger-dtx** [comment]
One more thing:
```
2021/02/03 05:43:29 listen tcp :10999: bind: address already in use
```

we have seen this for clusters on Microsoft Azure. 
> The root cause of this issue is that kubenet is not used in your AKS cluster. However, it is needed to retrieve the podCidr according to the official docs: https://docs.microsoft.com/en-us/rest/api/aks/managedclusters/createorupdate#containerservicenetworkprofile

**Comment by christian-kreuzberger-dtx** [comment]
I'm closing this due to lack of responses. If the problem persists, please re-open this issue.

**Comment by zzxwill** [comment]
> The installation hanging problem has already been solved in #2988 and will be fixed in 0.8.0.

Is there a nighty-build release for me to retry right now? Thanks.

**Comment by christian-kreuzberger-dtx** [comment]
You can try https://github.com/keptn/keptn/releases/tag/0.8.0-dev.202102191239

Download a CLI binary, and install it on your cluster using
```
keptn install --chart-repo=https://github.com/keptn/keptn/releases/download/0.8.0-dev.202102191239/keptn-installer-0.8.0-dev.tgz [--use-case=continuous-delivery]
```
